build.sbt

```diff
@@ -2,7 +2,7 @@ enablePlugins(JDKPackagerPlugin, JavaAppPackaging, SolidityPlugin)
 
 val commonSettings = Seq(
   name := "mantis",
-  version := "1.0-cli-rc1",
+  version := "1.0-cli",
```

Comment:
Do we need to update somewhere in `Daedalus` where Mantis is launched as well?

Reply:
Actually this branch is totally separated from branch integrating with `Daedalus`  so no changes necessary